### PR TITLE
chore: set team-k8s as maintainers

### PR DESCRIPTION
**Pull request by pmalek**
#### What this PR does / why we need it:

Set team-k8s@konghq.com as maintainers

### CODE REVIEW

**Review by programmer04 — DISMISSED** [review]

File: charts/ingress/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ description: Deploy Kong Ingress Controller and Kong Gateway
 home: https://konghq.com/
 icon: https://s3.amazonaws.com/downloads.kong/universe/assets/icon-kong-inc-large.png
 maintainers:
-  - name: mheap
-    email: michael.heap@konghq.com
+  - name: team-k8s-bot
```
Comment (czeslavo) [inline charts/ingress/Chart.yaml:6]: Why not:
```suggestion
  - name: team-k8s
```

File: charts/kong/Chart.yaml [hunk 2]
```diff
@@ -3,10 +3,8 @@ description: The Cloud-Native Ingress and API-management
 home: https://konghq.com/
 icon: https://s3.amazonaws.com/downloads.kong/universe/assets/icon-kong-inc-large.png
 maintainers:
-  - name: hbagdi
-    email: harry@konghq.com
-  - name: rainest
-    email: traines@konghq.com
+  - name: team-k8s-bot
```
Comment (czeslavo) [inline charts/kong/Chart.yaml:6]:
```suggestion
  - name: team-k8s
```

File: charts/ingress/Chart.yaml (hunk 1 above)
Comment (pmalek) [reply charts/ingress/Chart.yaml:6]: that's why https://github.com/Kong/charts/actions/runs/6550547949/job/17789894947#step:7:49

```
Error: failed linting charts: failed processing charts
------------------------------------------------------------------------------------------------------------------------
 ✖︎ ingress => (version: "0.7.0", path: "charts/ingress") > failed validating maintainer "team-k8s": 404 Not Found
 ✖︎ kong => (version: "2.29.0", path: "charts/kong") > failed validating maintainer "team-k8s": 404 Not Found
```

**Review by rainest — APPROVED** [review]
I figure the email and chart repo matter more than the username, so if the linter wants an actual GitHub user, whatever, placate it.

**Pull request merged**